docs/auth-modes.md

```diff
@@ -276,15 +276,15 @@ To enable the "Impersonate using embedded password" option in the publish dialog
     </customizations>
  ```
 
-The connector plugin present on a Tableau Server instance MUST contain the above two code changes and the server admin should run "tsm pending-changes apply". After this, user should be able to use a vanilla plugin on a Tableau Desktop Client with the same class name and capable of doing basic username-password authentication. Thereafter, when the user tries to publish a workbook using Tableau Desktop to a Tableau Server instance, they should be able to publish it using the option "Impersonate using embedded password". When the workbook is opened in Tableau Server, the delegation should have taken place automatically if appropriate delegation permissions are configured for the authenticated_user and delegated_user(user logged into Tableau server) or else an error message of this sort is thrown from the database "User user1 is not authorized to delegate to user2". For more information on deploying connector plugins in Tableau Server, please refer this [document]({{ site.baseurl }}/docs/run-taco)
+The connector plug-in present on a Tableau Server instance MUST contain the above two code changes and the server admin should run "tsm pending-changes apply". After this, user should be able to use a vanilla plugin on a Tableau Desktop Client with the same class name and capable of doing basic username-password authentication. Thereafter, when the user tries to publish a workbook using Tableau Desktop to a Tableau Server instance, they should be able to publish it using the option "Impersonate using embedded password". When the workbook is opened in Tableau Server, the delegation should have taken place automatically if appropriate delegation permissions are configured for the authenticated_user and delegated_user(user logged into Tableau server) or else an error message of this sort is thrown from the database "User user1 is not authorized to delegate to user2". For more information on deploying connector plug-ins in Tableau Server, please refer this [document]({{ site.baseurl }}/docs/run-taco)
```

Comment:
Changing "Connector Plugin" to "Connector Plug-in" is inconsistent with the rest of the SDK, including the readme, samples, error messages, and in some commands. Plus, the KB article for the Connector SDK uses "plugin". Even if the hyphen is technically more correct here, I'd vote for keeping "plugin" with no hyphen, since it's widely used elsewhere in the industry and it's consistent with how we've spelled it elsewhere and in the past.

Reply:
KB articles using "Connector Plugin":
https://help.tableau.com/current/pro/desktop/en-us/basicconnectoverview.htm
https://help.tableau.com/current/pro/desktop/en-us/examples_connector_sdk.htm